src/thread.c

```diff
@@ -270,6 +291,9 @@ static void *thread_pool_func(void *opaque)
 
 uint32_t MTY_ThreadPoolDispatch(MTY_ThreadPool *ctx, MTY_AnonFunc func, void *opaque)
 {
+	if (!ctx)
```

Comment:
should we either emit an MTY_Log warning specific to this new error (NULL context being passed) or emit the same MTY_Log warning we do when we can't find an available index?